docs/getting_started/contract_manage.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Contract management
+sidebar_position: 12
+---
+
+Contract management is a crucial feature that allows users to effectively interact with installed and deployed contracts. This feature enables users to upload their own `.wasm` and `ABI` files to the Console and deploy them to the blockchain.
+
+When you navigate to the Contract Management Page, you will be presented with two tabs: **Installed Contracts** and **Deployed Contracts**.
+
+![](../../static/img/contract_management.png)
+
+
+## Installed Contract
+
+Installed contract page shows the list of installed contracts for your specific shard. Installing the contract means putting the contract in the Installed state, meaning uploading the contract files to GCS and making the contract available for deployment from the Calimero Console. To install your contract:
+
+- Clicking on the **Install contract** button, which will prompt you to upload your contract files.
+- Provide a unique contract name, which will be displayed in the list of deployable contracts and used for storage purposes.
+- Upload your binary file (.wasm)
+- Upload the ABI file. If you upload an ABI file, a preview block will display the contents of the ABI file.
+- Click on the **Install new contract**.
+
+![](../../static/img/install_contract.png)
+
+## Deployed Contract
+
+![](../../static/img/deployed_contract-page.png)
+
+On the Deployed Contracts tab, you can see a list of contracts that have been deployed on selected shard. Once a contract is installed, it can be deployed to blockchain from the deploy contract page. 
+To deploy your contract on the blockchain, follow these steps:
+
+- Select the installed contract that you want to deploy.
+- Add your contract ID.
+- Click on **Create account and deploy contract** button
+
+![](../../static/img/deployed-contract.png)
+
+This action also creates a subaccount, which can be viewed under [custodial accounts](/docs/getting_started/custodial.md), and deployes the smart contract wasm to it.
```

Comment:
This action will create a subaccount[custodial accounts](/docs/getting_started/custodial.md) and deploy the smart contract .wasm to it.

Inline .wasm

@frdomovic can you please consult on this

Reply:
This action will create a new account (which will be a subaccount of shard master account) and this account will be used for smart contract deployment. 
-something like this i guess?